config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -69,13 +69,13 @@ periodics:
         - "./scripts/ci-e2e.sh"
       env:
         - name: KUBERNETES_VERSION_UPGRADE_FROM
-          value: "stable-1.19"
-        - name: KUBERNETES_VERSION_UPGRADE_TO
           value: "stable-1.20"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "stable-1.21"
         - name: ETCD_VERSION_UPGRADE_TO
           value: "3.4.13-0"
         - name: COREDNS_VERSION_UPGRADE_TO
-          value: "1.7.0"
+          value: "v1.8.0"
```

Comment:
Are these coredns upgrades just to test that out (just thinking that there's nothing in particular about a specific patch version of coredns that is especially compatible with a particular minor release of Kubernetes)

Reply:
the diff is rendering weird because it shifted all the jobs by 1 but all I did here was remove the 1.18-1.19 job and add the 1.22-1.23 job. coredns upgrade versions are aligned with CAPI upgrade tests https://github.com/kubernetes/test-infra/blob/66b80578bed131ac2091020bc513d7bb60f5a337/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml#L208

Reply:
https://github.com/coredns/deployment/blob/master/kubernetes/CoreDNS-k8s_version.md